### DISCUSSION TITLE
Question: Spanish models

### DISCUSSION THREAD

**Discussion by lasfito** [winkjs/wink-nlp#96] [New Features & Ideas]
Hi, 
so reading the [documentation](https://winkjs.org/wink-nlp/language-models.html) it seems like there are only two English models as of right now.  Browsing through the repo issues it seems like there was Portuguese support in the works. 

Is Spanish anywhere in the road map? 
Is there any way I could help contribute to that? (not an expert on the topic, though)

Thanks!

**Comment by rachnachakraborty** [comment]
Hi @lasfito,

We appreciate your interest in contributing to winkNLP. 

While we have ambitious plan for language models but we have been slow in model development. It will be great if you could contribute. 

One idea that we are currently exploring is to build some kind of universal model, which will support multiple languages such as French, German, Spanish and Hindi. However it will have limited pipeline options. 

What do you think of such an approach?

Would be happy to take it forward.

Best,
Rachna
